src/components/NewsRoomCard.jsx

```diff
@@ -0,0 +1,55 @@
+import {
+  Card,
+  CardContent,
+  CardFooter,
+  Button
+} from "@baltimorecounty/dotgov-components";
+import React from "react";
+
+const NewsRoomCard = props => {
+  const {
+    publishDate,
+    title,
+    author,
+    articleSummary,
+    thumbnail,
+    thumbnailAltText,
+    url
+  } = props;
+
+  const options = {
+    year: "numeric",
+    month: "long",
+    day: "numeric"
+  };
+
+  const published = new Date(publishDate).toLocaleDateString("en-US", options);
+
+  const imageSection = thumbnail ? (
```

Comment:
I think we should considr this a component which would change this slighltyl.

```jsx
const NewsThumbnail = ({thumbnail, thumbnailAltText}) => thumbnail ? <img ... : <i ...;
```

This way we can easily move it out of this file if needed

this would also change how you call it, but i tink thats ok and makes it more clear that this contains some markup.

```jsx
<NewsThumbnail thumbnail={thumbnail} alt={thumbnailAltTetxt} />
```

this is just an example and not tested, but you get the idea